llvm.5.0.0: failed to install with LLVM_EXPERIMENTAL_TARGETS_TO_BUILD options

`opam install llvm.5.0.0` is failed when llvm has already been built with options like `-DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD="WebAssembly"`.

In this case, `llvm-config --targets-built` becomes to print `WebAssembly` in their outputs. However https://github.com/ocaml/opam-repository/blob/master/packages/llvm/llvm.5.0.0/files/install.sh#L14 uses these outputs into `-DLLVM_TARGETS_TO_BUILD` as is. Thus `opam install llvm.5.0.0` shows errors as below.

```
### stderr ###
# [...]
# + cmake -DCMAKE_BUILD_TYPE=Release '-DLLVM_TARGETS_TO_BUILD=X86;WebAssembly' -DLLVM_OCAML_EXTERNAL_LLVM_LIBDIR=/usr/local/lib -DBUILD_SHARED_LIBS=FALSE -DLLVM_OCAML_OUT_OF_TREE=TRUE -DLLVM_OCAML_INSTALL_PATH=/root/.opam/4.04.0/lib ..
# CMake Error at CMakeLists.txt:688 (message):
#   The target `WebAssembly' does not exist.
# 
#       It should be one of
# 
#   
#   AArch64;AMDGPU;ARM;BPF;Hexagon;Lanai;Mips;MSP430;NVPTX;PowerPC;Sparc;SystemZ;X86;XCore
# 
# 
```

It seems difficult to distinguish which targets are experimental form outputs of `llvm-config --targets-built`. 

Thanks.

--- a/packages/llvm/llvm.5.0.0/files/install.sh
+++ b/packages/llvm/llvm.5.0.0/files/install.sh
@@ -5,13 +5,17 @@ libdir="$2"
 cmake="$3"
 make="$4"
 
+function filter_experimental_targets {
+    sed 's/AVR//g' | sed 's/Nios2//g' | sed 's/WebAssembly//g' | xargs
+}
+
 function llvm_install {
     mkdir build
     cd build
 
     "$cmake" \
         -DCMAKE_BUILD_TYPE="`"$llvm_config" --build-mode`" \
-        -DLLVM_TARGETS_TO_BUILD="`"$llvm_config" --targets-built | sed 's/ /;/g'`" \
+        -DLLVM_TARGETS_TO_BUILD="`"$llvm_config" --targets-built | filter_experimental_targets | sed 's/ /;/g'`" \
         -DLLVM_OCAML_EXTERNAL_LLVM_LIBDIR="`"$llvm_config" --libdir`" \
         -DBUILD_SHARED_LIBS=`[ $1 = "shared" ] && echo TRUE || echo FALSE` \
         -DLLVM_OCAML_OUT_OF_TREE=TRUE \

